examples/chat-example/Cargo.toml

```diff
@@ -10,5 +10,14 @@ async-std = { version = "1.12", features = ["attributes"] }
 async-trait = "0.1"
 env_logger = "0.10.0"
 futures = "0.3.27"
-libp2p = { path = "../../libp2p", features = ["async-std", "gossipsub", "mdns", "mplex", "noise", "macros", "tcp", "yamux"] }
+libp2p = { path = "../../libp2p", features = [
+    "async-std",
+    "gossipsub",
+    "mdns",
+    "noise",
+    "macros",
+    "tcp",
+    "yamux"
+] }
 libp2p-quic = { path = "../../transports/quic", features = ["async-std"] }
+libp2p-mplex = { path = "../../muxers/mplex" }
```

Comment:
I don't think we should show deprecated features in our example. Let's remove this!